oak-run/src/main/java/org/apache/jackrabbit/oak/run/Utils.java

```diff
@@ -178,7 +183,7 @@ public static NodeStore bootstrapNodeStore(NodeStoreOptions options, Closer clos
             System.exit(1);
         }
 
-        if (src.startsWith(MongoURI.MONGODB_PREFIX) || src.startsWith("jdbc")) {
+        if (src.startsWith(MONGODB_PREFIX) || src.startsWith("jdbc")) {
```

Comment:
maybe avoid unneeded change here?

Reply:
re-added in https://github.com/apache/jackrabbit-oak/pull/1459